### PR TITLE
Gh 342/topology info

### PR DESCRIPTION
**Pull request by victorsowa12**
add new serial message type, callback case, and unit test for bm_common_network_info_t

also bumping bm_common_messages

### CODE REVIEW

File: test/bm_serial_ut.cpp [hunk 1]
```diff
@@ -351,3 +351,44 @@ TEST_F(NCPTest, ConfigTest) {
...
+    .revision = 3,
+    .gitSHA = 1234,
+  };
+
+  uint64_t node_list[] = {12345678};
+  uint16_t num_nodes = 1;
+
+  uint16_t total_len = sizeof(bm_common_network_info_t) + num_nodes*sizeof(uint64_t);
+  bm_common_network_info_t *_network_info = (bm_common_network_info_t*)malloc(total_len);
+  _network_info->newtwork_crc32 = 1234;
```
Comment (russelldeguzman) [inline test/bm_serial_ut.cpp]: May be nice to make a convenience function like `bm_serial_pub` or `bm_serial_set_rtc`
to handle all the message packing code

**Review by uma-kat — APPROVED** [review]
approve

File: test/bm_serial_ut.cpp (hunk 1 above)
Comment (victorsowa12) [reply test/bm_serial_ut.cpp]: added in latest push, lmk what you think

Comment (victorsowa12) [reply test/bm_serial_ut.cpp]: ~I'm actually going to change it... we need one to pack the message together then one to send it.~ Leaving it as is, will move CRC32 calculation/checking to outside of bm_serial

File: bm_serial.c [hunk 2]
```diff
@@ -285,6 +285,45 @@ bm_serial_error_e bm_serial_set_rtc(bm_serial_time_t *time) {
...
+/*!
+  Send out a bm_common_network_info_t
+
+  \param[in] *config_crc
+  \param[in] *fw_info
+  \param[in] num_nodes
+  \param[in] *node_id_list
+  \return BM_SERIAL_OK if sent, nonzero otherwise
+*/
+bm_serial_error_e bm_serial_send_network_info(uint32_t network_crc32, bm_common_config_crc_t *config_crc, bm_common_fw_version_t *fw_info, uint16_t num_nodes, uint64_t* node_id_list) {
```
Comment (russelldeguzman) [inline bm_serial.c:297]: I would add a validity check for the pointers and break out if they're NULL

**Review by russelldeguzman — APPROVED** [review]
Small change ^ but otherwise looks good.

**Pull request merged**